### PR TITLE
added new parameter: availability zone

### PR DESCRIPTION
**Pull request by mavlyutov**

### CODE REVIEW

**Comment by mavlyutov** [comment]
summon @pupssman 
plz C

**Comment by pupssman** [comment]
Engaging..

File: openstack-cloud/src/main/java/jenkins/plugins/openstack/compute/JCloudsSlaveTemplate.java [hunk 1]
```diff
@@ -202,6 +204,11 @@ public NodeMetadata get() {
             options.as(NovaTemplateOptions.class).keyPairName(keyPairName);
         }
 
+        if (!Strings.isNullOrEmpty((availabilityZone))) {
```
Comment (pupssman) [inline openstack-cloud/src/main/java/jenkins/plugins/openstack/compute/JCloudsSlaveTemplate.java]: Missing good old lisp times?

**Comment by pupssman** [comment]
Looks nice)

**Pull request merged**